lib/ansible/modules/cloud/docker/docker_service.py

```diff
@@ -117,54 +107,46 @@
         - Images will only be rebuilt if Docker detects a change in the Dockerfile or build directory contents.
         - Use the C(nocache) option to ignore the image cache when performing the build.
         - If an existing image is replaced, services using the image will be recreated unless C(recreate) is I(never).
-      type: bool
       required: false
```

Comment:
It's *somewhat* clear that all these are truthy values, but IMHO we should make that explcit *somehow*.